From 5b698ba3faffd4e198a45be9fe74f53307395e4b Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fvogt@suse.de>
Date: Wed, 7 Apr 2021 13:38:09 +0200
Subject: [PATCH] Remove invalid g_object declarations to fix build with glib
 >= 2.68

g_object_ref_sink is defined as a macro meanwhile and so the build fails.
Just remove the declarations, glib.h is included anyway.

(Backported to 5.212. The glib include was added later, so the description
didn't match yet.)
---
 Source/WTF/wtf/glib/GRefPtr.h | 3 ---
 1 file changed, 3 deletions(-)

Index: qtwebkit-5.212.0-alpha4/Source/WTF/wtf/glib/GRefPtr.h
===================================================================
--- qtwebkit-5.212.0-alpha4.orig/Source/WTF/wtf/glib/GRefPtr.h
+++ qtwebkit-5.212.0-alpha4/Source/WTF/wtf/glib/GRefPtr.h
@@ -28,9 +28,7 @@
 #include <wtf/GetPtr.h>
 #include <wtf/RefPtr.h>
 #include <algorithm>
-
-extern "C" void g_object_unref(gpointer);
-extern "C" gpointer g_object_ref_sink(gpointer);
+#include <glib.h>
 
 namespace WTF {
 
